### DISCUSSION TITLE
resolved

### DISCUSSION THREAD

**Discussion by VetroX1** [netbox-community/netbox-docker#886] [Getting Help] [answered]
--

**Comment by kkthxbye-code** [comment, marked as answer]
You are changing the logo for the docs, not for netbox itself. Should be `/opt/netbox/netbox/static/netbox_logo.svg`. Please note that it's not a supported modification though.

**Reply by VetroX1** [reply]
Thanks for info, but is it in the right line of the compose tough?

**Reply by kkthxbye-code** [reply]
Yes, looks fine from a quick glance.

**Reply by VetroX1** [reply]
thank you very much
